jina/peapods/pods/k8slib/kubernetes_tools.py

```diff
@@ -11,6 +13,9 @@
     cur_dir, '..', '..', '..', 'resources', 'k8s', 'template'
 )
 
+if False:
+    from contextlib import GeneratorContextManager
```

Comment:
For which python version `GeneratorContextManager` exists? I can find a `_GeneratorContextManager`, but not this.

Reply:
I use python3.7

Reply:
changed to Generator thanks